### ISSUE TITLE
[ONNX] Will symbolic of custom autograd.Function be deprecated?

### ISSUE THREAD

**Issue by grimoire** [pytorch/pytorch#90054]
### 🚀 The feature, motivation and pitch

We are used to add custom symbolic follow this test:
https://github.com/pytorch/pytorch/blob/6f4dea562de0a1388f3d63b5ae1ecc15a47d254f/test/onnx/test_custom_ops.py#L66-L75

When I turn to PyTorch >=1.13, export such custom op would give me logs:
```
FutureWarning: 'torch.onnx._patch_torch._graph_op' is deprecated in version 1.13 and will
 be removed in 1.14. Please note 'g.op()' is to be removed from torch.Graph.
```

Did that means symbolic in the `autograd.Function` would be deprecated? Or will the graph be wrapped into a `GraphContext`?

Besides, error happened when I am trying to export op below:
```python
class CustomAdd(torch.autograd.Function):

    @staticmethod
    def forward(ctx, x, y):
        return x + 0

    @staticmethod
    def symbolic(g, x, y):
        return g.op('domain::CustomAdd', x, y)

```
with log:
```
    _C._jit_pass_peephole(graph, True)
IndexError: Argument passed to at() was not in the map.
```

I am not sure if it is related to the deprecation.

**Comment by janeyx99** [comment]
Hi! We tend to use GitHub issues only for bug reports or feature requests. Please use the forum to discuss these questions: [ https://discuss.pytorch.org/](https://discuss.pytorch.org/)

**Comment by BowenBao** [comment]
No, `symbolic of custom autograd.Function` will not be deprecated. @justinchuby feel free to chime in to comment on updates to `g.op`.

**Comment by justinchuby** [comment]
It will be wrapped into GraphContext. You can assume the g.op() interface but not g.other_things
